### PR TITLE
Nikita/Move the check box "Terms and Conditions to next page (Page 2)" 

### PR DESCRIPTION
**Pull request by nikita-binary**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_signup-affiliate.js [hunk 1]
```diff
@@ -68,87 +70,77 @@ const SignupAffiliate = ({
...
-            </LoginText>
-        </StyledFlex>
-    )
-}
+                    {localize('Want to sign up as a trader?')}
+                    <LocalizedLinkText to="/signup">
+                        <StyledLinkText id="dm-new-login-button" size="14px" color="grey-5">
+                            {' '}
+                            Create a Deriv account
+                        </StyledLinkText>
```
Comment (matin-binary) [inline src/components/custom/_signup-affiliate.js:96]: Can we use `Link` only for the text and exclude the spacing?

Comment (nikita-binary) [reply src/components/custom/_signup-affiliate.js:96]: Yes, will do refactor here.

File: src/components/custom/_signup-affiliate.js [hunk 2]
```diff
@@ -68,87 +71,75 @@ const SignupAffiliate = ({
...
+            {localize('Enter your email address to begin')}
+        </Header>
+        {!is_ppc && (
+            <StyledNote>
+                <Header
+                    as="p"
+                    type="paragraph-2"
+                    weight="normal"
+                    align="center"
+                    notedBox
```
Comment (sean-binary) [inline src/components/custom/_signup-affiliate.js:89]: What does `notedBox` do here?

Comment (nikita-binary) [reply src/components/custom/_signup-affiliate.js:89]: Yes, no need for that prop here. Will remove it.

**Review by matin-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**